dapps/marketplace/src/pages/growth/WithEnrolmentModal.js

```diff
@@ -545,8 +545,10 @@ function withEnrolmentModal(WrappedComponent) {
 
       return (
         <Query query={profileQuery} notifyOnNetworkStatusChange={true}>
-          {({ error, data }) => {
-            if (error) {
+          {({ error, data, networkStatus, loading }) => {
+            if (networkStatus === 1 || loading) {
+              return null
+            } else if (error) {
```

Comment:
@nick @sparrowDom Just a note: It was throwing some error randomly (i.e. when data is still loading), Had to add this to fix those tests. But I have no change in this PR that could have caused this. Either way, we should probably change all queries to check the networkStatus before accessing the data.